qiita_pet/templates/redbiom.html

```diff
@@ -11,22 +11,17 @@
     $("#redbiom-table").dataTable({
       "dom": '<"ps">lfr<t><"footer">ip',
       'footerCallback': function ( row, data, start, end, display ) {
+        var api = this.api(), artifacts = 0, samples = 0;
 
-        // adding total features found
+        api.column(0).nodes().each(function (cell, i) {
+          var elements = $(cell).children().children().children();
+          var idx = elements.length - 1;
+          var text = $(elements[idx]).text();
+          elements = text.split(" | ");
+          artifacts += parseInt(elements[0].split(": ")[1]);
+          samples += parseInt(elements[1].split(": ")[1]);
```

Comment:
what's actually being summed here? the number of samples in a study or artifacts?

Reply:
Excellent question. In the code below (lines 48-56) we count the number of artifacts and samples per study (row). Here we are building the footer to show the total so we could recount all studies/artifacts/samples or just loop over the totals (each row) and parse out the numbers to add. The format of each row is: 'Artifacts: ## |  Samples: ##'.